firmware/arduino_libraries/PID/PID.h

```diff
@@ -68,7 +80,7 @@ typedef struct
 
 
 
-void pid_update( PID* pid, double curr_error, double dt );
+int pid_update( PID* pid, double curr_error, double dt );
```

Comment:
Adding PID_ERROR and PID_SUCCESS makes sense, I can incorporate similar defines for overrides in throttle.